src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/InviteUsers/InviteOrganizationUsersCommand.cs

```diff
@@ -206,9 +206,25 @@ await sendOrganizationInvitesCommand.SendInvitesAsync(
 
     private async Task SendAdditionalEmailsAsync(Valid<InviteOrganizationUsersValidationRequest> validatedResult, Organization organization)
     {
+        await SendNotifyOwnersOfAutoscaleIncreaseAsync(validatedResult, organization);
         await SendPasswordManagerMaxSeatLimitEmailsAsync(validatedResult, organization);
     }
 
+    private async Task SendNotifyOwnersOfAutoscaleIncreaseAsync(Valid<InviteOrganizationUsersValidationRequest> validatedResult, Organization organization)
```

Comment:
I think a better name (and possible xmldoc) could make the method more explicit
```suggestion
    private async Task NotifyOwnersIfAutoscaleIncreasesSeatCountAsync(Valid<InviteOrganizationUsersValidationRequest> validatedResult, Organization organization)
```